user/languages/javascript-with-nodejs.md

```diff
@@ -121,6 +121,54 @@ npm install
 
 Note that `npm install` can fail if a shrinkwrapped git dependency pointing to a branch has its HEAD changed.
 
+## Ember Apps
+You can build your Ember Apps on Travis CI and can automate test. By default ember comes with [`Qunit`](http://qunitjs.com/) as testing framework. This is an example using different ember versions.
+
```

Comment:
Ember by default has qunit . While serving and also doing test it automatically build distribution version . The quint here is to give further reference to those who want to learn more about it .